### PR TITLE
1 opened

### PR DESCRIPTION
**Pull request by wanda-carlson**
Opened by wanda

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # public repo
+new
```
Comment (henry-richard) [inline README.md:2]: lgtm

**Review by henry-richard — COMMENTED** [review]
lgtm

**Review by henry-richard — CHANGES_REQUESTED** [review]
message

**Review by henry-richard — APPROVED** [review]
message

**Review by henry-richard — COMMENTED** [review]
message

**Comment by wanda-carlson** [comment]
hey

**Pull request merged**